bool ignoreMe
---
nav_msgs/Path path